Podfile.lock

```diff
@@ -35,7 +35,7 @@ DEPENDENCIES:
 
 SPEC CHECKSUMS:
   Representor: 487b4efb0a39c2c2db258d40846d9915cd69b770
-  Result: 6c990ec4a72470672f9fc5b6fef009da0f6f40d1
+  Result: af24e3e57ab81005ec4221a53fd36dc1e7a52869
```

Comment:
Just to point out, this change is due to the podspec for `Result` in CocoaPods was updated.